genesis/ext/pyrender/jit_render.py

```diff
@@ -360,8 +360,9 @@ def load_programs(self, renderer, flags, program_flags):
 
     def gen_func_ptr(self):
         self.gl = GLWrapper()
+        self.gl.build_wrapper()
 
-        IS_OPENGL_42_AVAILABLE = hasattr(self.gl, "glDrawElementsInstancedBaseInstance")
+        IS_OPENGL_42_AVAILABLE = "glDrawElementsInstancedBaseInstance" in self.gl.gl_funcs.keys()
```

Comment:
I’m surprise it did not work using ‘hasattr’. I forgot to check that before merging…

you could remove the extra ‘keys()’ to check if there is an item in a dictionary that is mapping the key you are looking for.